admin_ui/backend/export_agents_yaml.py

```
@@ -20,6 +20,14 @@ def export_yaml(store: AgentsStore) -> str:
         ctx = {"provider": a["provider"], "prompt": a["prompt"]}
         for k in ("voice", "greeting"):
             if a[k]: ctx[k] = a[k]
+        # LOW-A5: preserve operator metadata across an export -> delete -> re-migrate
+        # cycle. These aren't first-class YAML context keys, so on re-import they land
+        # in extra_json (harmless to the engine) rather than being silently dropped.
+        for k in ("extension", "role_label", "notes", "email_recipient", "email_from"):
+            if a[k]: ctx[k] = a[k]
```

Ruff (0.15.17)

[error] Multiple statements on one line (colon)

(E701)